addOns/reports/src/test/java/org/zaproxy/addon/reports/ExtensionReportsUnitTest.java

```diff
@@ -1669,6 +1713,51 @@ void shouldGenerateValidXmlPlusReport() throws Exception {
         assertThat(root.getAttribute("generated").length(), is(greaterThan(20)));
     }
 
+    @Test
+    void shouldGenerateValidXmlPlusReportWithStats() throws Exception {
+        // Given
+        Template template = getTemplateFromYamlFile("traditional-xml-plus");
+        String fileName = "basic-traditional-xml-plus";
+        File f = File.createTempFile(fileName, template.getExtension());
+        DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+        dbf.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true);
+        DocumentBuilder db = dbf.newDocumentBuilder();
+
+        InMemoryStats stats = new InMemoryStats();
+        ExtensionStats extStats =
+                mock(ExtensionStats.class, withSettings().strictness(Strictness.LENIENT));
+        given(extensionLoader.getExtension(ExtensionStats.class)).willReturn(extStats);
+        given(extStats.getInMemoryStats()).willReturn(stats);
+
+        stats.counterInc("http://example.com", "site.a", 1);
+        stats.counterInc("http://example.com", "site.b", 2);
+        stats.counterInc("global.x", 3);
+        stats.counterInc("global.y", 4);
+
+        // When
+        File r = generateReportWithAlerts(template, f);
+        Document doc = db.parse(r);
+        Element root = doc.getDocumentElement();
+
+        // Then
+        checkXmlAlert(doc, true);
+        assertThat(root.getNodeName(), is(equalTo("OWASPZAPReport")));
```

Comment:
I guess the "OWASP" is staying for backward compatibility?

Reply:
Yep